common/src/main/java/apoc/result/IndexConstraintRelationshipInfo.java

```diff
@@ -9,16 +9,19 @@ public class IndexConstraintRelationshipInfo {
 
     public final String name;
 
-    public final Object type;
+    public final String type;
 
     public final List<String> properties;
 
     public final String status;
+    
+    public final Object relationshipType;
 
-    public IndexConstraintRelationshipInfo(String name, Object type, List<String> properties, String status) {
+    public IndexConstraintRelationshipInfo(String name, String type, List<String> properties, String status, Object relationshipType) {
```

Comment:
can we call this `constraintType` then it is even less confusing, if you agree, change the one in Nodes as well :)

Reply:
Yes, changing the name is a good idea, but maybe I would change the name to `schemaType`, 
because `constraintType` = `INDEX` is personally confusing anyway.

It would seem clearer to me even if this change will be made: https://github.com/neo4j/apoc/pull/247#pullrequestreview-1188008107

Reply:
ah fair enough, schemaType works better :) just internally anyway